index.html

```diff
@@ -291,30 +316,46 @@ <h3>File Names</h3>
+                <p><dfn id="dfn-css-resource" data-lt="MiniApp CSS resource|MiniApp CSS resources" data-dfn-type="dfn">CSS resources</dfn> are stylesheets that describe the rendering and appearance of [=MiniApp pages=]. CSS resources can be global, affecting all the pages in a MiniApp (i.e., <code>app.css</code> in the [=root directory=]), or with local scope, just affecting a specific [=page=] in the MiniApp.</p>
+                <p>This specification supports the CSS modules as defined in the [[CSS-SNAPSHOT]].</p>
                 <p>CSS resources MUST meet all the following criteria:</p>
-                <dl class="conformance-list">
+                <dl class="conformance-list">                    
+                    <dt>Syntax and content</dt>
+                    <dd>A [=CSS resource=] MUST follow described using the syntax and definitions of CSS as described in the [[CSS-SNAPSHOT]].</dd>
+                    <dt>File name</dt>
+                    <dd>The global [=CSS resource=], affecting all [=pages=] in a MiniApp, MUST be named <code>app.css</code> and stored in the [=root directory=].</dd>
+                    <dd>A local [=CSS resource=] associated with a [=page=] SHOULD have a unique filename that identifies the [=MiniApp page=] and matches with the corresponding [=HTML resource=] and [=scripting resource=] if present.</dd>
                     <dt>File extension</dt>
-                    <dd>A MiniApp stylesheet filename SHOULD use the file extension <code>.css</code>.</dd>
+                    <dd>A [=CSS resource=] filename SHOULD use the file extension <code>.css</code>.</dd>
                 </dl>
             </section>
             <section id="sec-miniapp-resources-js">
                 <h3>Scripting Resources</h3>
-                <p>MiniApps contains scripting resources, documents written in JavaScript, that provide the MiniApp with functionality and logic.</p>
+                <p><dfn id="dfn-js-resource" data-lt="MiniApp scripting resource" data-dfn-type="dfn">Scripting resources</dfn> are documents that contain script elements and data to control the business logic and functionality of the application, adding interactivity and managing the lifecycle of the MiniApp. There is a global [=scripting resource=] (i.e., <code>app.js</code> in the [=root directory=]) that includes data and functions available for all the [=pages=] of the MiniApp, and with specific logic to control the application's lifecycle. Each [=MiniApp page=] MAY have associated specific [=scripting resources=], with the scope limited to the concrete [=page=].</p>
                 <p>Scripting resources MUST meet all the following criteria:</p>
                 <dl class="conformance-list">
+                    <dt>Syntax and content</dt>
+                    <dd>The [=scripting resources=] MUST conform with the requirements of the JavaScript specification [[JAVASCRIPT]].</dd>
```

Comment:
The "JavaScript" spec in the WHATWG is [obsolete](https://blog.whatwg.org/javascript). We should probably change it to the ECMAScript Language Specification `[ECMA-262]`.

Reply:
Done in bdc7276d17d32e75f14100dc6af93055409498fc